ramlfications/types.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015 Ramlfications contributors
+
+from __future__ import absolute_import, division, print_function
+import attr
+import copy
+from six import MAXSIZE
+from six import iteritems
+from six import string_types
+
+from ramlfications.errors import UnknownDataTypeError
+from ramlfications.errors import DataTypeValidationError
+from ramlfications.parameters import Content
+from ramlfications.utils.common import OrderedDict
+from ramlfications.utils.parser import convert_caml_case
```

Comment:
I'd put imports from the same module on the same lines, and use relative imports e.g.:

```python
from six import iteritems, string_types, MAXSIZE

from .errors import DataTypeValidationError, UnknownDataTypeError
from .parameters import Content
from .utils.common import OrderedDict
from .utils.parser import convert_caml_case
```